src/main/java/zeenea/connector/property/PropertyDefinition.java

```diff
@@ -171,18 +187,19 @@ public static TagPropertyDefinition tag(String code) {
    * @param label the label of the property definition
    * @return a new TagPropertyDefinition instance
    */
+  @Deprecated(since = "2.3.0", forRemoval = true)
```

Comment:
Pareil : 
```suggestion
  Et là également : 
```suggestion
  
  @Deprecated(
      since =
          "Deprecated since version 2.3.0, use tag(name) instead. Scheduled for removal in version 3.0.0.",
      forRemoval = true)
```
```